pkg/route/api/v1/types.go

```diff
@@ -41,7 +41,11 @@ type RouteSpec struct {
 
 	// To is an object the route points to. Only the Service kind is allowed, and it will
 	// be defaulted to Service.
-	To kapi.ObjectReference `json:"to"`
+	To RouteTargetReference `json:"to"`
+
+	// AdditionalTos is an extension of the 'to' field. If more than one service needs to be
+	// pointed to, then use this field
+	AdditionalTos []RouteTargetReference `json:"additionalTos,omitempty"`
```

Comment:
And just coincidentally `alternateBackends` can be abbreviated `A/B`, which if you ask me is bankable.